CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# How to contribute
+
+Operator Marketplace is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, instructions on how to set up a dev environment, and other resources to help get contributions into operator-marketplace.
+
+# Email and Chat
+
+- Email: [operator-framework](https://groups.google.com/forum/#!forum/operator-framework)  
+
+## Setting up a dev environment
+
+- Fork the repository on GitHub
+- Clone the repository in your go path.
+- Get access to an Openshift cluster. The marketplace-operator is currently supported on Openshift 4.0. You can find instructions on installing an Openshift cluster [here](https://github.com/openshift/installer).
+- Turn off the Cluster-Service-Operator (CVO). The CVO manages the lifecycle of the marketplace-operator along with other internal Openshift components. Once the CVO is truned off, the marketplace-operator deployment can be deleted. 
+    `oc scale --replicas 0 -n openshift-cluster-version deployments/cluster-version-operator`
+- Delete the marketplace-operator deployment `oc delete deployment openshift-marketplace -n marketplace-operator`
+- Install [Operator-SDK](https://github.com/operator-framework/operator-sdk).
+- `cd` into the marketplace-operator repository and `dep ensure` to download all dependencies.
```

Comment:
Why is this needed? We should have everything in the `vendor` folder.

Reply:
@aravindhp to build yes, but for whatever reason the sdk does not read those dependencies locally. at least that is what we found on friday when trying to get this to run in Anik's local environment.

Reply:
That should be called out here. This should also be brought up with the SDK team.

Reply:
@kevinrizza I think that might have been due to the fact that my cloned repo was not in my go path.

Reply:
@anik120 did you confirm with a clean environment?